tests/test_load_quotes.py

```diff
@@ -0,0 +1,22 @@
+from app.database.models import Quote
+from app.internal.quotes import load_quotes
+
+
+# Tests for loading the quotes in the db:
+def test_load_daily_quotes(session):
+    load_quotes.load_daily_quotes(session)
+    assert session.query(Quote).count() > 0
```

Comment:
assert is_quotes_table_empty is False

Reply:
I thought it's better not to use an inner function of the module, for the actual testing of that module...
However, I will add separate count function for all the tests that use the count (in the tests file).